package.json

```diff
@@ -26,10 +26,9 @@
     "gulp-less": "1.3.6",
     "gulp-print": "1.1.0",
     "gulp-replace": "0.5.0",
+    "gulp-run": "1.6.6",
     "gulp-wrap": "0.5.0",
-    "gulp-wrap-amd": "0.3.1",
     "jshint-stylish": "1.0.0",
-    "requirejs": "2.1.15",
     "run-sequence": "1.0.2",
     "streamqueue": "0.1.1"
   }
```

Comment:
Missing webpack/gulp-webpack here, I had to install webpack manually from npm to make it compile.
If you use gulp-webpack you can probably remove the gulp-run again.